app/views/play/cinematic/cinematicController.js

```diff
@@ -64,17 +79,88 @@ export class CinematicController {
   async startUp () {
     const data = await this.systems.loader.loadAssets()
 
-    // Todo: Intentionally only parse a single shot for Goal 1.
-    const commands = parseShot(data.shots[0], this.systems)
+    const commands = data.shots
+      .map(shot => parseShot(shot, this.systems))
 
     // TODO: There must be a better way than an array of locks! In future add reasonable timeout with `Promise.race`.
     await Promise.all(this.startupLocks)
 
     attachListener({ cinematicLankBoss: this.systems.cinematicLankBoss, stage: this.stage })
 
-    const runner = new CommandRunner(commands)
-    await runner.run()
+    // This is hot start. We don't need to start immediately.
```

Comment:
Question to clarify whether I understand this comment correctly: 

Does this mean that it will start the same way (not `await`'ing), but we don't have to block on the return of this function? 

If so, perhaps the wording could be tweaked slightly to indicate that? The `We don't need to start immediately` gives me the impression that the actual `startUp` will happen later.